gcsstore.go

```diff
@@ -159,6 +159,31 @@ func (g *GcsFS) listObjects(q *storage.Query, retries int) (Objects, error) {
 	return nil, lasterr
 }
 
+// Folders
+func (g *GcsFS) Folders(ctx context.Context, csq Query) ([]string, error) {
+	var q = &storage.Query{Delimiter: csq.Delimiter, Prefix: csq.Prefix}
```

Comment:
Can we default the delimiter to `/`?  That way it's an advanced feature and simpler to use?